### ISSUE TITLE
Implement CI/CD and package publishing

### ISSUE THREAD

**Issue by joelzamboni** [wearewebera/openvela#3]
The current workflow for publishing the PyPi package is disabled because it was just the template, and the package has been published manually. Let's create a workflow that implements it correctly using GitHub Actions

**Comment by SamuelSPatrocinio** [comment]
@joelzamboni I need the pypi log in

**Comment by SamuelSPatrocinio** [comment]
Using the example @joelzamboni set up, I created the CI/CD workflow in the `pypi.yml` file. It builds the package, publishes to pypi and creates a release on GitHub. The information on how to trigger this workflow and create a new release is in the repository's `readme` file. To test, I released a new version (v0.1.2) and It successfully built and published to pypi. It also generated a release on GitHub.


![Image](https://github.com/wearewebera/openvela/assets/73202930/bbdac286-5228-4527-ba11-fc37553ac7cd)
